integtest/src/main/java/org/corfudb/universe/cluster/docker/CorfuServerDockerized.java

```diff
@@ -0,0 +1,324 @@
+            Thread.currentThread().interrupt();
+            throw new NodeException("Can't stop Corfu server", e);
+        }
+    }
+
+    /**
+     * Immediately kill the Corfu server.
+     *
+     * @throws NodeException this exception will be thrown if the server can not be killed.
+     */
+    @Override
+    public void kill() throws NodeException {
+        log.info("Killing the Corfu server. {}", params.getGenericName());
+
+        try {
+            docker.killContainer(params.getGenericName());
+        } catch (DockerException | InterruptedException ex) {
+            Thread.currentThread().interrupt();
+            throw new NodeException("Can't kill Corfu server", ex);
+        }
+    }
+
+    /**
+     * Deploy and start docker container, expose ports, connect to a network
+     *
+     * @return docker container id
+     */
+    private String deployContainer() throws NodeException {
+        ContainerConfig containerConfig = buildContainerConfig();
+
+        String id;
+        try {
+            ContainerCreation creation = docker.createContainer(containerConfig, params.getGenericName());
+            id = creation.id();
+
+            addShutdownHook(id);
+
+            docker.connectToNetwork(id, docker.inspectNetwork(clusterParams.getNetworkName()).id());
+
+            docker.startContainer(id);
+        } catch (InterruptedException | DockerException e) {
+            Thread.currentThread().interrupt();
+            throw new NodeException("Can't start a container", e);
+        }
+
+        return id;
+    }
+
+    private ContainerConfig buildContainerConfig() throws NodeException {
+        // Bind ports
+        String[] ports = {String.valueOf(params.getPort())};
+        Map<String, List<PortBinding>> portBindings = new HashMap<>();
+        for (String port : ports) {
+            List<PortBinding> hostPorts = new ArrayList<>();
+            hostPorts.add(PortBinding.of("0.0.0.0", port));
+            portBindings.put(port, hostPorts);
+        }
+
+        HostConfig hostConfig = HostConfig.builder()
+                .portBindings(portBindings)
+                .autoRemove(true)
+                .build();
+
+        // Compose command line for starting Corfu
+        String cmdLine = String.format(
+                "java -cp *.jar org.corfudb.infrastructure.CorfuServer %s",
+                getCommandLineParams()
+        );
+
+        try {
+            return ContainerConfig.builder()
+                    .hostConfig(hostConfig)
+                    .image("corfu-server:" + getAppVersion())
+                    .hostname(params.getGenericName())
+                    .exposedPorts(ports)
+                    .cmd("sh", "-c", cmdLine)
+                    .build();
+        } catch (XmlPullParserException | IOException e) {
+            throw new NodeException("Can't parse application version", e);
+        }
+    }
+
+    private void addShutdownHook(String id) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "id". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)